pkg/fqdn/name_manager_gc.go

```diff
@@ -241,7 +240,7 @@ func (n *NameManager) RestoreCache(preCachePath string, restoredEPs []EndpointDN
 			alive, _ := possibleEP.DNSZombies.GC()
 			for _, zombie := range alive {
 				for _, name := range zombie.Names {
-					n.cache.Update(lookupTime, name, []netip.Addr{zombie.IP}, int(2*DNSGCJobInterval.Seconds()))
+					n.cache.Update(lookupTime, name, []netip.Addr{zombie.IP}, int(2*defaults.DNSGCJobInterval.Seconds()))
```

Comment:
@pippolo84 Should we change this to `activeConnectionTTL` ?

Reply:
Yeah that makes sense to me, that would match the equivalent `Update()` calls in `GC()`.